### PR TITLE
Fix check

### PR DESCRIPTION
**Pull request by mfordjody**

### CODE REVIEW

File: content/zh-cn/overview/tasks/observability/tracing/zipkin.md [hunk 1]
```diff
@@ -151,4 +151,4 @@ public class KafkaSenderConfiguration {
 
 ### SpringBoot2案例
 
-dubbo-tracing相关的使用在SpringBoot2与3中区别不大，SpringBoot2的案例可参考[代码地址](https://github.com/apache/dubbo-samples/tree/master/4-governance/dubbo-samples-spring-boot-tracing)。
+dubbo-tracing相关的使用在SpringBoot2与3中区别不大，SpringBoot2的案例可参考[代码地址](https://github.com/apache/dubbo-samples/tree/master/4-governance/dubbo-samples-spring-boot3-tracing)。
```
Comment (AlbumenJ) [inline content/zh-cn/overview/tasks/observability/tracing/zipkin.md:154]: @conghuhu PTAL

**Review by AlbumenJ — CHANGES_REQUESTED** [review]

File: content/zh-cn/overview/tasks/observability/tracing/zipkin.md (hunk 1 above)
Comment (conghuhu) [reply content/zh-cn/overview/tasks/observability/tracing/zipkin.md:154]: Due to recent adjustments to the dubbo-samples directory related to tracing, I haven't had time to make any changes on the dubbo website yet.
The correct url for SpringBoot2 zipkin sample now should be [https://github.com/apache/dubbo-samples/tree/master/4-governance/dubbo-samples-tracing/dubbo-samples-spring-boot-tracing-zipkin]( https://github.com/apache/dubbo-samples/tree/master/4-governance/dubbo-samples-tracing/dubbo-samples-spring-boot-tracing-zipkin)

Comment (AlbumenJ) [reply content/zh-cn/overview/tasks/observability/tracing/zipkin.md:154]: > Due to recent adjustments to the dubbo-samples directory related to tracing, I haven't had time to make any changes on the dubbo website yet. The correct url for SpringBoot2 zipkin sample now should be https://github.com/apache/dubbo-samples/tree/master/4-governance/dubbo-samples-tracing/dubbo-samples-spring-boot-tracing-zipkin

Cloud you pls submit a PR to fix this page?

Comment (conghuhu) [reply content/zh-cn/overview/tasks/observability/tracing/zipkin.md:154]: Okay, I will organize the documents for tracing today

Comment (mfordjody) [reply content/zh-cn/overview/tasks/observability/tracing/zipkin.md:154]: thanks！

**Pull request closed without merging**